client/app/states/services/details/details.html

```diff
@@ -234,10 +234,10 @@ <h2 translate>Virtual Machines ({{ ::vm.vms.length }})</h2>
               </span>
             </div>
             <div class="col-lg-1 col-md-1 hidden-sm hidden-xs">
-              <button class="btn btn-default open-console-button" ng-if="item['supports_console?'] && item['supports_vnc_console?'] && item.power_state == 'on'" tooltip="{{'HTML5 console'|translate}}" tooltip-placement="bottom">
+              <button class="btn btn-default open-console-button" ng-if="!item.supported_consoles.vnc.visible && !item.supported_consoles.vnc.enabled" tooltip="{{'HTML5 console'|translate}}" tooltip-placement="bottom">
```

Comment:
@AllenBW looks like this is now missing that `&& item.power_state == 'on'` ?

(Assuming we still don't want to run consoles on powered-off VM's..)

Reply:
@jntullo correct me if i'm wrong... 🤔

Reply:
@AllenBW you're correct. `enabled` ensures that it is powered on

Reply:
OK, then the only remaining problem ... the condition is negative again :D ;)

Reply:
I assume `ng-if="item.supported_consoles.vnc.visible && item.supported_consoles.vnc.enabled"`